db/migrate/20200430072846_create_invited_users.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateInvitedUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :invited_users do |t|
+      t.integer :user_id
```

Comment:
Can this column be empty? Do we need a `null: false` check here? Also we should probably add an index on this column as well.

Reply:
Yes, `user_id` is empty initially, as per: https://github.com/discourse/discourse/pull/9813/files#diff-2a355fe6078d7a0ee0e440532e4032e2R102

Will add index.